lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMPass.cpp

```diff
@@ -569,6 +582,7 @@ struct ConvertTritonGPUToLLVM
            CacheKeyDenseMapInfo>
       indexCache;
   mlir::triton::gpu::TMAMetadataTy *tmaMetadata = nullptr;
+  std::map<std::string, int> lowerStats;
```

Comment:
Why is this called "lowerStats" here rather than pStat?  Is it something different?